test/suites/standard/reboot.robot

```diff
@@ -0,0 +1,36 @@
+*** Settings ***
+Documentation       Tests related to system reboot
+
+Resource            ../../resources/common.resource
+Resource            ../../resources/oc.resource
+Resource            ../../resources/ostree-health.resource
+
+Suite Setup         Setup Suite With Namespace
+
+Test Tags           restart    slow
+
+
+*** Variables ***
+${SOURCE_POD}           ./assets/reboot/pod-with-pvc.yaml
+${POD_NAME_STATIC}      test-pod
+
+
+*** Test Cases ***
+Reboot Test
+    [Documentation]    Run a reboot test and make sure pod with PVC remains functional.
+    [Setup]    Test Case Setup
+    Reboot MicroShift Host
+    Wait Until Greenboot Health Check Exited
```

Comment:
Let's change this to `Wait For Healthy System`?

Reply:
this keyword is shared with  different suits, i wouldn't want to refactor them all in this PR, we can open refactor PR later.